tests/rptest/tests/license_enforcement_test.py

```diff
@@ -201,3 +201,27 @@ def test_enterprise_cluster_bootstrap(self, root_driven_bootstrap):
                                  timeout_sec=60,
                                  backoff_sec=1,
                                  err_msg="The cluster hasn't stabilized")
+
+    @cluster(num_nodes=5)
```

Comment:
5 nodes seems excessive.

Reply:
agreed - the test class actually uses 5 RP nodes, I can make a new class and use only three if you prefer

Reply:
> agreed - the test class actually uses 5 RP nodes, I can make a new class and use only three if you prefer

Do any of the tests need 5 brokers?

Reply:
> > agreed - the test class actually uses 5 RP nodes, I can make a new class and use only three if you prefer
> 
> Do any of the tests need 5 brokers?

Not mine.  I didn't want to touch the original ones